rotkehlchen/chain/aggregator.py

```diff
@@ -923,9 +923,6 @@ def query_evm_chain_balances(self, chain: SUPPORTED_EVM_CHAINS) -> None:
         chain_balances = self.balances.get(chain)
         queried_balances = manager.node_inquirer.get_multi_balance(accounts)
         for account, balance in queried_balances.items():
-            if balance == ZERO:
-                continue
-
```

Comment:
this makes more sense but in the case of the account having 0 ETH I believe you are adding a 0 to the balances. Would be better to create an empty `BalanceSheet` since we don't return 0 balances. Also add a comment explaining why you do it

Reply:
the way i did it, I'm not sure if it needed a comment